docs/DevelopmentGuide.md

```diff
@@ -45,16 +41,38 @@ The test suite uses [RSpec](https://rspec.info/) tests to verify the correctness
 
 ### Writing tests
 
-New tests should be written as RSpec tests in the `spec/ddtrace` folder. Test files should generally mirror the structure of `lib`.
+New tests should be written as RSpec tests in the `spec/datadog` folder. Test files should generally mirror the structure of `lib`.
 
 All changes should be covered by a corresponding RSpec tests. Unit tests are preferred, and integration tests are accepted where appropriate (e.g. acceptance tests, verifying compatibility with datastores, etc) but should be kept to a minimum.
 
 **Considerations for CI**
 
-All tests should run in CI. When adding new `spec.rb` files, you may need to add a test task to ensure your test file is run in CI.
+All tests should run in CI. When adding new `_spec.rb` files, you may need to add rake task to ensure your test file is run in CI.
+
+ - Ensure that there is a corresponding Rake task defined in `Rakefile` under the `spec` namespace, whose pattern matches your test file. For example
+
+ ```
+   namespace :spec do
+     RSpec::Core::RakeTask.new(:foo) do |t, args|
```

Comment:
do you think we would want to add `desc ""` before the rake task in the example?

Reply:
I left this out because I think it is trivial since it only affects the display from `rake -T` .